vtex-apps/search-result
# feat: add facets default to the shipping filter

#### What problem is this solving?
Show delivery facets even when CEP is not defined.
[SAX-334](https://vtex-dev.atlassian.net/browse/SAX-334)

#### How to test it?

Search for a product (example query: "air"), without a defined CEP, and see the Shipping filter in the sidebar.

[Workspace](https://arthurdp--fulfillmentqa.myvtex.com/air?_q=air&fuzzy=0&initialMap=ft&initialQuery=air&map=ft&operator=and)



#### Screenshots or example usage:
<img width="271" alt="image" src="https://github.com/user-attachments/assets/5f2a9771-2a79-4e2c-9ecc-d7db2eec3a5e">

#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://i.gifer.com/tiJ.gif)
